### PR TITLE
Adding robot vacuum commands

### PR DESCRIPTION
**Pull request by raww**
These are the vacuum commands - looking at adding support for switchbot robot vacuum cleaners to homeassistant.

source: https://github.com/OpenWonderLabs/SwitchBotAPI#robot-vacuum-cleaner-s1-2

### CODE REVIEW

**Comment by raww** [comment]
@SeraphicRav would love to get this merged - I've got another one to follow which has webhooks added.

**Comment by raww** [comment]
And webhooks for switchbot wired into ha.

File: switchbot_api/__init__.py [hunk 1]
```diff
@@ -142,6 +142,15 @@ class LightCommands(Commands):
     BRIGHTNESS_DOWN = "brightnessDown"
 
 
+class VacuumCommands(Commands):
+    """Vacuum commands."""
+    
+    START = "start"
+    STOP = "stop"
+    DOCK = "dock"
+    FAN_SPEED = "PowLevel"
```
Comment (SeraphicRav) [inline switchbot_api/__init__.py]: I think we should try to be close to the vocabulary used by Switchbot, can we rename `FAN_SPEED` into `POWER_LEVEL` ?

**Review by SeraphicRav — CHANGES_REQUESTED** [review]
@raww Thank you very much for your pull request ! I am eager to see your next one.

I hope you did not wait too long for your PR, I did not receive any notification, I will add myself as a code owner 😃

File: switchbot_api/__init__.py (hunk 1 above)
Comment (raww) [reply switchbot_api/__init__.py]: Yeah you're right, I was in HA mode and they call it fan speed.

**Comment by SeraphicRav** [comment]
Thanks!

**Pull request merged**